src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx

```diff
@@ -32,7 +32,7 @@ type EndingOverlayProps = {
   isEnded: boolean
 }
 // 10 seconds
-const NEXT_VIDEO_TIMEOUT = 10000
+const NEXT_VIDEO_TIMEOUT = 10 * 1000
```

Comment:
Why did you make this change?

Reply:
just to make it clearer that it is 10 secs and easier to change later